src/components/chip/chip.tsx

```diff
@@ -92,7 +92,7 @@ export class Chip
   /** When `true`, the icon will be flipped when the element direction is right-to-left (`"rtl"`). */
   @Prop({ reflect: true }) iconFlipRtl = false;
 
-  /** Specifies the size of the component. */
+  /** Specifies the size of the component. When contained in a parent `calcite-chip-group` inherits the parent's `scale` value and displays consistent spacing. */
```

Comment:
Maybe leave off the spacing portion here since it is set at the chip group level.